django_cas_ng/backends.py

```diff
@@ -232,7 +233,7 @@ def _verify_cas2_saml(ticket, service):
 _verify = _PROTOCOLS[settings.CAS_VERSION]
 
 
-class CASBackend(object):
+class CASBackend(ModelBackend, object):
```

Comment:
Maybe take `object` out of the inheritance because it's the base for `ModelBackend`?